team-builders/src/components/FormCard.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { Container, Row, Col, Card, CardBody, CardText } from "reactstrap";
```

Comment:
the form looks great. Prebuilt components are great for a quick project like this, in my opinion.